mayland-blocks/block-template-parts/header.html

```diff
@@ -8,22 +8,20 @@
 <!-- /wp:column -->
 
 <!-- wp:column {"verticalAlignment":"center","className":"is-vertically-aligned-bottom"} -->
-<div class="wp-block-column is-vertically-aligned-center is-vertically-aligned-bottom">
```

Comment:
I'm seeing the site title is misaligned compared to the navigation block. I'm going to push a little change so they are centered.